### ISSUE TITLE
Are there an equivalent to Vimium's next page command?

### ISSUE THREAD

**Issue by gonejack** [flipxfx/sVim#73]
<img width="883" alt="qq20171001-131542 2x" src="https://user-images.githubusercontent.com/2720676/31052020-279c2a40-a6ab-11e7-9096-c56c94f8d9f8.png">
This command should be an easy feature that looks for the link elements containing special words and just bind them to the dedicate key but quite useful, could you add it for people author?

**Comment by abrookins** [comment]
This has been on my short list to add for many months. Whenever I have some free time I'll take a look -- but I don't have much time lately!

**Comment by isundaylee** [comment]
I'd love to take a stab at this feature, but just wanna make sure that I'm not duplicating any existing work. @abrookins Do you mind if I take a stab at this? :)

**Comment by nikitavoloboev** [comment]
This extension must be rewritten for Safari 12.0 first. It doesn’t work in Mojave’s.

**Comment by isundaylee** [comment]
I'd love to help with the rewrite! But I probably would need some time ramping up on the new Safari extension. Also my Apple Developer Account just expired and I'm still deciding whether it makes sense for me to renew.

Meanwhile, I implemented a basic version of this feature (#97). Can that be merged so that this feature can be used for pre-Mojave Safari? I tested it as a local build in Safari's "Extension Builder", but couldn't build the `.safariext` file since I don't have a valid Apple Developer Account anymore.

Thanks! :)
